front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -523,23 +555,77 @@ export default function AssistantBuilder({
             </div>
           }
           rightPanel={
-            <div
-              className={classNames(
-                "flex h-full w-full overflow-hidden rounded-xl border border-structure-200 bg-structure-50 transition-all",
-                shouldAnimatePreviewDrawer &&
-                  previewDrawerOpenedAt != null &&
-                  // Only animate the reload if the drawer has been open for at least 1 second.
-                  // This is to prevent the animation from triggering right after the drawer is opened.
-                  Date.now() - previewDrawerOpenedAt > 1000
-                  ? "animate-reload"
-                  : ""
+            <div className="h-full pb-5">
+              {template ? (
+                <Tab
+                  tabs={previewDrawerTabs}
+                  variant="default"
+                  className="hidden lg:flex"
+                />
+              ) : null}
+              {previewDrawerCurrentTab === "Preview" ? (
+                <div
+                  className={classNames(
+                    "flex h-full w-full overflow-hidden rounded-xl border border-structure-200 bg-structure-50 transition-all",
+                    shouldAnimatePreviewDrawer &&
+                      previewDrawerOpenedAt != null &&
+                      // Only animate the reload if the drawer has been open for at least 1 second.
+                      // This is to prevent the animation from triggering right after the drawer is opened.
+                      Date.now() - previewDrawerOpenedAt > 1000
+                      ? "animate-reload"
+                      : ""
+                  )}
+                >
+                  <TryAssistant
+                    owner={owner}
+                    assistant={draftAssistant}
+                    conversationFading={isFading}
+                  />
+                </div>
+              ) : (
+                <ReactMarkdown
```

Comment:
will maybe try to DRY this with message rendering later on @flvndvd

Reply:
Can we at least extract it in a component in this file?

Reply:
WIll do separately, I think we want markdown component that can already render everything so we can use in both chat and here